src/test/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtilsUnitTest.java

```diff
@@ -144,6 +144,32 @@ public void testQueryWithNullLabel() {
         checkQueryResults(result, getAllExpectedNamesAndAges(), query);
     }
 
+    @Test(groups = {"cloud"})
+    public void testQueryWithNullWorkflow_Name() { // THIS IS THE TEST IM WRITING
+        final Map<String, String> expectedNamesAndAges = getAllExpectedNamesAndAges();
+        final String query = String.format("SELECT * FROM `%s`", BIGQUERY_FULLY_QUALIFIED_TABLE);
+        final List<String> fieldsToRetrieve = new LinkedList<>();
+        final String noUDFs = null;
+        fieldsToRetrieve.add("name");
+        Map<String, String> labels = new HashMap<String, String>();
```

Comment:
nitpick to minimize Java syntax noise
```suggestion
        Map<String, String> labels = new HashMap<>();
```